infrastructure/tests/test_search_page.py

```diff
@@ -298,5 +298,4 @@ def test_search_events(self):
         # Remove filters with a dropdown menu
         self.click("#functions-dropdown")
         self.click("#functions-dropdown")
```

Comment:
Why `self.click("#functions-dropdown")` twice?

Reply:
The first click expands the dropdown and the second click selects the first option which is 'All Government Functions' to remove the filter

Just above this, the test expands the dropdown and then selects a specific project function to filter by